webview.h

```diff
@@ -2123,12 +2190,12 @@ class win32_edge_engine {
     return true;
   }
 
-  void resize(HWND wnd) {
+  void resize_widget() {
```

Comment:
I think I understand why a window handle was passed here—we could receive `WM_SIZE` before setting `GWLP_USERDATA` which would cause this function to be called on a `nullptr`. Now we handle `WM_NCCREATE` which eliminates that problem.